transforms/code/code2parquet/python/Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN useradd -ms /bin/bash dpk
 USER dpk
 WORKDIR /home/dpk
 
+# Create directories to mount volumnes for processing data outside of the image.
+RUN mkdir input && mkdir output
+
```

Comment:
I am not convinced we need this. According to https://www.warp.dev/terminus/docker-run-volume you can mount any local directory into a Docker container and then use it for execution. There is no need to do mkdir. Can you please verify this. Even more info is here https://docs.docker.com/storage/volumes/

Reply:
Ok yes, no need to create the mount points ahead of time.  Reverted the Dockerfiles and updated the transform readmes to point to the quickstart for running images.  Thanks.